### PR TITLE
[CUMULUS-508] Enable multiple azs and ensure docker uses devicemapper storage driver

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
**Summary:** 

Addresses [CUMULUS-508: Cumulus should allow for lambdas and ECS clusters to have multiple AZ availability](https://bugs.earthdata.nasa.gov/browse/CUMULUS-508)

## Changes

* Enable multiple azs with additional template configuration `ecs.availabilityZones`
* Ensure docker uses devicemapper storage driver. The NGAP 2.0 AMI uses the `overlay2` storage driver by default, while cumulus-sndbx ami's were using `devicemapper` by default. In [the docker docs](https://docs.docker.com/storage/storagedriver/select-storage-driver/) it says overlay2 is preferred but I wasn't (yet) able to get the disk space size configuration to work with overlay2.

## Test Plan
Things that should succeed before merging.
- [x] Adhoc testing in the NGAP 2.0 account with NGAP 2.0 AMIs
- [x] Testing in the cumulus-sndbx account with existing AMIs
- [x] Update CHANGELOG

### CODE REVIEW

File: packages/deployment/app/cloudformation.template.yml [hunk 1]
```diff
@@ -777,6 +777,7 @@ Resources:
           Fn::Join:
             - ""
             - - "#cloud-boothook\n"
+              - "mkdir -p /etc/docker && echo '{\n  \"storage-driver\": \"devicemapper\"\n}' >> /etc/docker/daemon.json\n"
```
Comment (scisco) [inline packages/deployment/app/cloudformation.template.yml]: what does this do?

File: packages/deployment/app/cloudformation.template.yml [hunk 2]
```diff
@@ -790,7 +791,13 @@ Resources:
         MinInstancesInService: 0
     Properties:
       AvailabilityZones:
+      {{# if ecs.availabilityZones }}
```
Comment (scisco) [inline packages/deployment/app/cloudformation.template.yml]: I like the fact that this change preserve backward compatibility 👏 💯

Comment (scisco) [reply packages/deployment/app/cloudformation.template.yml]: Is there anything we should update the docs?

File: packages/deployment/app/cloudformation.template.yml (hunk 1 above)
Comment (abarciauskas-bgse) [reply packages/deployment/app/cloudformation.template.yml]: This enables the `devicemapper` storage driver https://docs.docker.com/storage/storagedriver/device-mapper-driver/

File: packages/deployment/app/cloudformation.template.yml (hunk 2 above)
Comment (abarciauskas-bgse) [reply packages/deployment/app/cloudformation.template.yml]: good idea, opened https://github.com/cumulus-nasa/cumulus-nasa.github.io/pull/55/

**Pull request merged**